### ISSUE TITLE
ARMLA breaks when chromosomes differ

### ISSUE THREAD

**Issue by robinweide** [robinweide/GENOVA#184]
**Describe the bug**
APA (and possibly other ARMLA-members cannot deal with two experiments with different (amounts of) chromosomes. 

**To Reproduce**
explist_10k contains to WT-datasets: 1 juicebox and 1 hicpro. When I run APA on them seperately, it doesn't give errors.

The juicebox-contacts has this info:
```
-  CHRS	:	A vector of 24 chromosome names.
- CENTROMERES:	Locations of 24 centromeres.
```

The Hicpro-contacts has this info:
```
-  CHRS	:	A vector of 24 chromosome names.
- CENTROMERES:	Locations of 23 centromeres.
```

 APA_out <- APA(explist = explist_10k[1:2], bedpe = loops)

 Error in vapply(seq_along(explist)[-1], function(i) { : 
  values must be type 'logical',
 but FUN(X[[1]]) result is type 'character'

**note**
The data for reproducing this is found in projects/GENOVA/[obscured]_matrices.Rdata

**Comment by teunbrand** [comment]
The error sounds like #173 , but I thought we made that error message more informative by #174 . Is your local GENOVA version up to date with master or dev? (Should have been fixed/more informative with both).

With regards to the actual error; yes ARMLA functions check wether the IDX's of all samples are identical, if they are not, they will throw an error as they use 1 set of anchor indices to go through the data of all samples.

I see 2 potential solutions for this:
1. Introduce a synchronisation function for the .hic derived samples, therewith also adressing Angela's comment on #165 .
2. Do the ARMLA functions seperately on each sample and the use bundle/unbundle functions to combine them for visualisation/quantification.
